controllers/istiocni/istiocni_controller_test.go

```diff
@@ -440,7 +440,7 @@ func TestApplyImageDigests(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			result := applyImageDigests(tc.input, tc.input.Spec.Values, tc.config)
+			result := applyImageDigests(tc.input.Spec.Version, tc.input.Spec.Values, tc.config)
```

Comment:
This invokes `applyImageDigests` with the unresolved version name instead of the version number as in the production code. Is this okay?